ClientApp/src/App.jsx

```diff
@@ -5,190 +5,191 @@ import DefaultLayout from "./layouts/Default";
 //import UserService from "./components/services/userService";
 import "./utils/toastr.css";
 import {
-   authProtectedFlattenRoutes,
```

Comment:
These files should not have this many corrections outside your feature's scope. If this is a "code cleanup" tool, then refrain from using it unless it is on the 'CODE BLOCK" you worked on. The reason being is everyone could have different code cleanups, and if you code clean up every time you push, this will cause mass merge conflicts. The only files that should report changes are the files you worked on. If another file has been modified that you did not directly work on, you should ideally undo all git changes to that file.